packages/ra-core/src/controller/list/useListParams.ts

```diff
@@ -364,6 +364,10 @@ export const getQuery = ({
         query.page = 1;
     }
 
+    if (!query.filter || Object.keys(query.filter).length === 0) {
+        query.filter = filterDefaultValues || {};
```

Comment:
now we use filterdefaultValues even if there are some params - but only if the filter params is empty.